cuda_core/tests/test_event.py

```diff
@@ -15,8 +17,17 @@
 def test_timing(init_cuda, enable_timing):
     options = EventOptions(enable_timing=enable_timing)
     stream = Device().create_stream()
-    event = stream.record(options=options)
-    assert event.is_timing_disabled == (not enable_timing if enable_timing is not None else True)
+    n_seconds = 0.5
+    e1 = stream.record(options=options)
+    time.sleep(n_seconds)
+    e2 = stream.record(options=options)
+    for e in (e1, e2):
+        assert e.is_timing_disabled == (not enable_timing if enable_timing is not None else True)
```

Comment:
This is a brain teaser!

How about

```
    assert e.is_timing_disabled == (True if enable_timing is None else enable_timing)
```

Reply:
AFK but quick, I think I'm missing a `not`

Reply:
Ok edit from phone worked 😅

should be correct now

or

not bool(enable_timing)

maybe, although that's less readable

Reply:
fixed in commit 8aca468